src/crypto/PSAOperationalKeystore.cpp

```diff
@@ -209,5 +252,56 @@ void PSAOperationalKeystore::ReleasePendingKeypair()
     mIsPendingKeypairActive = false;
 }
 
+CHIP_ERROR PSAOperationalKeystore::MoveOpKeysFromKVSToITS(PersistentStorageDelegate * storage)
+{
+    Crypto::SensitiveDataBuffer<TLV::EstimateStructOverhead(sizeof(uint16_t), P256SerializedKeypair::Capacity())> buf;
+    uint16_t keySize = static_cast<uint16_t>(buf.Capacity());
+
+    for (FabricIndex index = 0; index < CHIP_CONFIG_MAX_FABRICS; index++)
```

Comment:
This is wrong. Fabric index can be any value between kMinValidFabricIndex and kMaxValidFabricIndex, so it can be bigger than CHIP_CONFIG_MAX_FABRICS. I think one option is not to call this function from `Init`, but from Server after initializing FabricTable (and then get the list of existing fabrics from FabricTable).